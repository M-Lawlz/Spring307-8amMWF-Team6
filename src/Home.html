<h1> Welcome to SimTrek </h1>