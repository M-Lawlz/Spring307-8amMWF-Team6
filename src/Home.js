import React from "react";

export default class Home extends React.Component {
  render() {

    return (

<div  className="hi">
	<h1> Welcome to SimTrek!</h1>
</div>
)
    	
  }


}
