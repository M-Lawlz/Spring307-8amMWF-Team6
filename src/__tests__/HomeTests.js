import Home from "../Home";
import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

test("Home renders successfully", () => {
  const home = shallow(<Home />);
  expect(toJson(home)).toMatchSnapshot();
});
