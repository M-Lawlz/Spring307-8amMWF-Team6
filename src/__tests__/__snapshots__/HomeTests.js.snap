// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders successfully. 1`] = `
<div
  className="hi"
>
  <h1>
     Welcome to SimTrek!
  </h1>
</div>
`;
